sklearn/neighbors/_graph.py

```diff
@@ -351,6 +352,11 @@ class KNeighborsTransformer(
     (178, 178)
     """
 
+    _parameter_constraints = {
+        **NeighborsBase._parameter_constraints,
+        "mode": [StrOptions({"distance", "connectivity"})],
+    }
+
```

Comment:
Currently `NeighborsBase._parameter_constraitns` contains a constraint for `radius`, although it probably shouldn't, and it is causing issues here. For now you can just remove that constraint.
```suggestion
    _parameter_constraints = {
        **NeighborsBase._parameter_constraints,
        "mode": [StrOptions({"distance", "connectivity"})],
    }
    _parameter_constraints .pop("radius")    
```

Reply:
On to it

Reply:
@Micky774  updated _graph.py to remove constraint from radius